onert-micro/eval-driver/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(SRCS_EVAL_TESTER Driver.cpp)
+
+add_executable(onert_micro_eval_driver ${SRCS_EVAL_TESTER})
+
+set(READER_SUFFIX "_driver")
```

Comment:
Could you add some comment here?
Like "This variable is needed to separate standalone interpreter libraries from the libraries used in driver"

Reply:
Done